vector/src/main/java/im/vector/app/features/home/room/list/RoomListModule.kt

```diff
@@ -18,10 +18,15 @@ package im.vector.app.features.home.room.list
 
 import dagger.Binds
 import dagger.Module
+import fr.gouv.tchap.features.home.room.list.TchapRoomListViewModel
+import fr.gouv.tchap.features.home.room.list.TchapRoomListViewModelFactory
 
 @Module
 abstract class RoomListModule {
 
     @Binds
     abstract fun providesRoomListViewModelFactory(factory: RoomListViewModelFactory): RoomListViewModel.Factory
+
+    @Binds
+    abstract fun providesTchapRoomListViewModelFactory(factory: TchapRoomListViewModelFactory): TchapRoomListViewModel.Factory
```

Comment:
Don't you think we should remove the previous bind related to RoomListViewModel?

Reply:
this factory is injected in RoomListViewModel. Without this binding, roomListViewModelFactory is unknown and the build fails.

Reply:
TchapRoomListViewModelFactory has been replaced by RoomListViewModelFactory